bokehjs/src/lib/models/ui/tooltip.ts

```diff
@@ -113,7 +121,12 @@ export class TooltipView extends UIElementView {
       this.render()
       this.after_render()
     })
-    this.on_change([content, closable, interactive], () => {
+    this.on_change(content, async () => {
+      await this._build_content()
+      this.render()
+      this.after_render()
+    })
+    this.on_change([closable, interactive], () => {
```

Comment:
Ideally this `on_change` call would need to change and thus `render/after_render` would need duplication, but there's race condition related to async and non-async handlers preventing this.

Reply:
I'm not following you here. Are you saying that if there weren't a race condition you would have to call both of these methods twice?

Reply:
I would like to be able to write this:
```ts
    this.on_change(content, async () => {
      await this._build_content()
    })
    this.on_change([content, closable, interactive], () => {
      this.render()
      this.after_render()
    })
```
Handlers are run in registration order.